lib/modules/commentTools.js

```diff
@@ -298,114 +298,132 @@ modules['commentTools'] = {
 				this.fetchTitle();
 			}
 
+			fastdom.write(function() {
+				$body.on('click', 'div.markdownEditor-wrapper a:not(.userTagLink)', function(e) {
```

Comment:
@ArrestedDevelopment, is there an advantage to attaching event handlers during the write cycle?  I'm wondering if I should emulate this or if you just got on a roll :p